<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessSettingsDialog</class>
 <widget class="QWidget" name="ProcessSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Processing Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/resources/icons/process_tools.png</normaloff>:/resources/icons/process_tools.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="restoreButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Restore</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="accessibleName">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">QTabWidget::pane { /* The tab widget frame */
    border-top: 2px solid #C2C7CB;
    position: absolute;
    top: -0.5em;
}

QTabWidget::tab-bar {
    alignment: center;
}

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 2px solid #C4C4C3;
    border-bottom-color: #C2C7CB; /* same as the pane color */
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 8ex;
    padding: 2px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
}

QTabBar::tab:selected {
    border-color: #9B9B9B;
    border-bottom-color: #C2C7CB; /* same as pane color */
}</string>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="preProcessingTab">
        <attribute name="title">
         <string>Pre-Processing</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="4" column="0">
          <widget class="QLabel" name="windowImage">
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>150</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QComboBox" name="windowComboBox">
           <item>
            <property name="text">
             <string>Hamming</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hann</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bartlet</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blackman</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Kaiser</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Gauss</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="useDC">
             <property name="text">
              <string>Use Offset filter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="usePreEnphasis">
             <property name="text">
              <string>Use PreEnphasis filter</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Windowing</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="timeAnalysisTab">
        <attribute name="title">
         <string>Time Analysis</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Short Time Analysis</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Pitch Estimation</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="pitchMethodCombobox">
             <item>
              <property name="text">
               <string>ACF</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>AMDF</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CEPSTRUM</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Min Pitch</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Max Pitch</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="minPitch">
             <property name="minimum">
              <number>80</number>
             </property>
             <property name="maximum">
              <number>5000</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="maxPitch">
             <property name="minimum">
              <number>80</number>
             </property>
             <property name="maximum">
              <number>5000</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>500</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Window shift:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Window time:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="windowShift">
             <property name="readOnly">
              <bool>false</bool>
             </property>
             <property name="keyboardTracking">
              <bool>false</bool>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>0.005000000000000</double>
             </property>
             <property name="maximum">
              <double>0.040000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.005000000000000</double>
             </property>
             <property name="value">
              <double>0.010000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="windowTime">
             <property name="readOnly">
              <bool>false</bool>
             </property>
             <property name="keyboardTracking">
              <bool>false</bool>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>0.020000000000000</double>
             </property>
             <property name="maximum">
              <double>0.100000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.005000000000000</double>
             </property>
             <property name="value">
              <double>0.030000000000000</double>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Buffer Time</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="bufferTime">
             <property name="readOnly">
              <bool>false</bool>
             </property>
             <property name="keyboardTracking">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="maximum">
              <number>10</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Parameter</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="shortTimeMethodCombobox"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Pitch Analysis</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>label_6</zorder>
        <zorder>line_2</zorder>
        <zorder>label_9</zorder>
        <zorder>line_3</zorder>
        <zorder></zorder>
       </widget>
       <widget class="QWidget" name="frequencyAnalysisTabAnalysis">
        <property name="accessibleName">
         <string/>
        </property>
        <attribute name="title">
         <string>Frequency Analysis</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_13">
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Processing Configuration</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QComboBox" name="spectrogramMethod">
           <item>
            <property name="text">
             <string>FFT</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>LPC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cepstrum</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Filter Bank</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <layout class="QGridLayout" name="gridLayout_21">
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Samples</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="samplesFFT">
             <property name="minimum">
              <number>256</number>
             </property>
             <property name="maximum">
              <number>4096</number>
             </property>
             <property name="singleStep">
              <number>256</number>
             </property>
             <property name="value">
              <number>512</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="orderCepstrum">
             <property name="minimum">
              <number>8</number>
             </property>
             <property name="maximum">
              <number>20</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Order Cepstrum</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Order LPC</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="orderLPC">
             <property name="minimum">
              <number>8</number>
             </property>
             <property name="maximum">
              <number>20</number>
             </property>
             <property name="value">
              <number>13</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Filter Banks</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="orderFilterBanks">
             <property name="minimum">
              <number>10</number>
             </property>
             <property name="maximum">
              <number>30</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="labelInfo">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line_5">
     <property name="styleSheet">
      <string notr="true">Line {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 2px solid #C4C4C3;
    border-bottom-color: #C2C7CB; /* same as the pane color */
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 8ex;
    padding: 2px;
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
